src/logic/calculate.js

```diff
@@ -0,0 +1,54 @@
+import operate from './operate';
+
+function calculate(calculatorData, buttonName) {
+  let { total, next, operation } = calculatorData;
+  const operators = ['+', 'x', '-', '÷'];
```

Comment:
**REQUIRED**
* Operate switches operation and uses plus, minus, times, div and modifier (turning the number into a percentage based number, e.g. 95 % == .95) functions ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png))
Let's include the modifier operator.

Reply:
linters errors fixed locally